Resources/Prototypes/_NF/Procedural/dungeon_configs.yml

```diff
@@ -433,3 +434,85 @@
         SouthEast: BrickTileWhiteInnerSe
         NorthWest: BrickTileWhiteInnerNw
         NorthEast: BrickTileWhiteInnerNe
+
+- type: dungeonConfig
+  id: SalvageOutpost
+  generator: !type:PrefabDunGen
+    roomWhitelist:
+      - SalvageOutpost
+    presets:
+      - Bucket
+      - Wow
+      - SpaceShip
+      - Tall
+  postGeneration:
+    - !type:CorridorPostGen
+      width: 3
+
+    - !type:DungeonEntrancePostGen
+      count: 2
+
+    - !type:RoomEntrancePostGen
+      entities:
+        - CableApcExtension
+        - AirlockMiningGlass
+
+    - !type:EntranceFlankPostGen
+      entities:
+        - Grille
+        - MiningWindow
+
+    - !type:ExternalWindowPostGen
+      entities:
+        - Grille
+        - MiningWindow
+
+    - !type:WallMountPostGen
+      spawns:
+        # Posters
+        - id: RandomPosterLegit
+          orGroup: content
+        - id: ExtinguisherCabinetFilled
+          prob: 0.2
+          orGroup: content
+        - id: RandomPainting
+          prob: 0.05
+          orGroup: content
+        - id: DefibrillatorCabinetFilled
+          prob: 0.1
+          orGroup: content
+
+    - !type:BoundaryWallPostGen
+      tile: FloorSteel
+      wall: WallMining
+      cornerWall: WallMining
```

Comment:
From my observations (which isn't backed up by statistics though) players when faced with a choice between dismantling a wall or digging some more in hopes of finding a window usually go for digging.
Also, yes, this is intended to be something that players can use their hoarded explosives.

But I guess I should add deconstruction graphs, because both windows and walls have high damage thresholds .

Reply:
Hm.  Guess I really don't do expeditions that often then.

Should probably be left for another PR, at any rate.